cpp/dolfinx/mesh/utils.h

```diff
@@ -118,20 +127,22 @@ std::vector<std::int32_t> locate_entities_boundary(
     const std::function<xt::xtensor<bool, 1>(const xt::xtensor<double, 2>&)>&
         marker);
 
-/// Compute the indices the geometry data for the vertices of the given
-/// mesh entities
+/// @brief Determine the indices in the geometry data for each vertex of
+/// the given mesh entities.
 ///
-/// @param[in] mesh Mesh
+/// @param[in] mesh The mesh
 /// @param[in] dim Topological dimension of the entities of interest
-/// @param[in] entity_list List of entity indices (local)
+/// @param[in] entities Entity indices (local) to compute the vertex
+/// geometry indices for
 /// @param[in] orient If true, in 3D, reorients facets to have
 /// consistent normal direction
-/// @return Indices in the geometry array for the mesh entity vertices, i.e.
-/// indices(i, j) is the position in the geometry array of the j-th vertex of
-/// the entity entity_list[i].
-xt::xtensor<std::int32_t, 2>
+/// @return Indices in the geometry array for the entity vertices. The
+/// shape is `(num_entities, num_vertices_per_entity)` and the storage
+/// is row-major. The index `indices[i, j]` is the position in the
```

Comment:
```suggestion
/// is row-major. The index `indices[i * num_vertices_per_entity + j]` is the position in the
```

Reply:
Less clear to me.

Reply:
But indices[i,j] assumes a 2D structure, while what we have is flat. I agree it is more verbose, but it correct

Reply:
That's why I give the shape and the storage layout (row-major) in the docstring.